src/Object/Api/Mastodon/Status.php

```diff
@@ -135,26 +136,26 @@ public function __construct(array $item, Account $account, Counts $counts, UserA
 
 		$this->uri = $item['uri'];
 		$this->url = $item['plink'] ?? null;
-		$this->replies_count = $counts->replies;
-		$this->reblogs_count = $counts->reblogs;
-		$this->favourites_count = $counts->favourites;
+		$this->replies_count = $reblogged ? 0 : $counts->replies;
```

Comment:
What is meaning of this change?

Reply:
I'll double check this since it has been awhile but I believe that was from me trying to emulate the data that is not filled in with the Mastodon API for reblogged posts that we were still populating.